packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -354,6 +362,26 @@ export const CLIENT_ERROR_CODE_TO_ERROR_PAYLOAD: Record<number, Partial<ClientEv
     category: 'signaling',
     fatal: true,
   },
+  [MISSING_ROAP_ANSWER_CLIENT_CODE]: {
+    errorDescription: ERROR_DESCRIPTIONS.MISSING_ROAP_ANSWER,
+    category: 'signaling',
+    fatal: true,
+  },
+  [DTLS_HANDSHAKE_FAILED_CLIENT_CODE]: {
+    errorDescription: ERROR_DESCRIPTIONS.DTLS_HANDSHAKE_FAILED,
+    category: 'media',
+    fatal: true,
+  },
+  [ICE_FAILED_WITHOUT_TURN_TLS_CLIENT_CODE]: {
+    errorDescription: ERROR_DESCRIPTIONS.ICE_FAILED_WITHOUT_TURN_TLS,
+    category: 'media',
+    fatal: true,
+  },
+  [ICE_FAILED_WITHOUT_TURN_TLS_CLIENT_CODE]: {
```

Comment:
this should be "with" not "without". I see you have a unit test for this, so not sure why it didn't fail, can you check that too?

Reply:
addressed